README.md

```diff
@@ -26,24 +26,24 @@ pip install jupyterhub-ltiauthenticator
 
 ## Using LTIAuthenticator
 
-### EdX
+### Common Configuration Settings
+
+Due to the fact that LTI 1.1 is an open standard LMS vendors that adhere to the LTI 1.1 standard utilize the same configuration settings when integrating with an external tool. Some of these settings are included in a configuration endpoint to facilitate the JupyterHub's as an external tool with your LMS.
 
-1.  You need access to [EdX Studio](https://studio.edx.org/) to set up LTI. You
-    might have to contact your EdX Liaison to get access.
+Start by following the steps below to configure your JupyterHub setup with the basic settings. Then, navigate to your LMS vendor's section to complete the installation and configuration steps.
 
-2.  [Enable LTI Components](http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/lti_component.html#enabling-lti-components-for-a-course)
-    for your course.
+**Note**: if you LMS is not listed feel free to send us a PR with instructions for this new LMS.
 
-3.  Create a _client key_ and _client secret_ for use by edX to authenticate your hub. Open up any command line:
+1.  Create a _client key_ and _client secret_ for use by edX to authenticate your hub. Open up any command line:
 
     1.  Run `openssl rand -hex 32` and save the output. This will be your LTI Client **Key**.
     2.  Run `openssl rand -hex 32` and save the output. This will be your LTI Client **Secret**.
 
-    Now we will use these strings to allow edX and JupyterHub to authenticate each other.
+Now we will use these strings to allow edX and JupyterHub to authenticate each other.
 
-    _Note_: These commands will simply generate strings for you, it will not store them anywhere on the computer. Therefore you do not need to run these commands on your JupyterHub server--we will be supplying them manually in the next few steps.
+_Note_: These commands will simply generate strings for you, it will not store them anywhere on the computer. Therefore you do not need to run these commands on your JupyterHub server--we will be supplying them manually in the next few steps.
 
-    _Note_: Anyone with these two strings will be able to access your hub, so keep them secure!!
+_Note_: Anyone with these two strings will be able to access your hub, so keep them secure!!
 
 4.  By default, the user's name will be the `custom_canvas_user_id` passed in by canvas. If you
```

Comment:
Numbering is now weird, you can use 1. at all time to let the rendered markdown figure out the numbers to avoid these kinds of troubles.